scripts/local-deploy.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+import "../src/CosmoShips.sol";
+import "../src/GameLeague.sol";
+import "../src/IAttributeVerifier.sol";
+import "../test/fixtures/mockVerifier.sol";
+import "../test/fixtures/mockRandomGenerator.sol";
+
+contract MyScript is Script {
```

Comment:
contract `Deploy` is Script.
Also, the file name should match the contract name